services/core-web/src/reducers/noticeOfWorkReducer.js

```diff
@@ -26,6 +28,16 @@ const noticeOfWorkReducer = (state = initialState, action) => {
         ...state,
         originalNoticeOfWork: action.payload,
       };
+    case actionTypes.STORE_NOTICE_OF_WORK_APPLICATION_REVIEWS:
+      return {
+        ...state,
+        noticeOfWorkReviews: action.payload.records,
+      };
```

Comment:
Are we wrapping everything in `records: {}` should it be 'reviews: {}' ?

Reply:
all list endpoints should return the results in the records member of the response object.